app/src/main/java/com/rootstrap/android/models/TargetModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.rootstrap.android.models
+
+import com.rootstrap.android.network.models.Target
+
+data class TargetModel(
```

Comment:
Don't use the Model suffix.

Reply:
I'm not sure why you need this TargetModel?

Reply:
I add the target model, because one is the target response, with the topic id, and this one has the reference to the Topic